test/integration/graphql_test.lua

```diff
@@ -2051,3 +2051,65 @@ g.test_propagate_defaults_to_callback = function()
     t.assert_equals(errors, nil)
     t.assert_items_equals(json.decode(data.prefix.test_mutation), result)
 end
+
+function g.test_float_variables()
+    local query = [[
+        query ($arg: Float) { test(arg: $arg) }
```

Comment:
It's relevant if float value passed via variables. Let's test also argument value.